lsst/ts/cbp/statemachine.py

```diff
@@ -513,6 +388,16 @@ def park(self):
         """
         self._cbp.set_park(1)
 
+    def connect(self):
+        self._cbp.connect()
+
+    def disconnect(self):
+        self._cbp.disconnect()
+
+    def configure(self, config):
+        self._cbp.config=config
+        self._cbp.configure()
```

Comment:
This is one place where passing `config` as an argument and inside self._dbp saving the config argument as attribute `config` pays off. It makes the API easier to use correctly and harder to use incorrectly because you cannot forget to call self._cbp.config=config before calling self._dbp.configure(). It also makes configure more uniform across the various objects in this package.